### DISCUSSION TITLE
converting to video

### DISCUSSION THREAD

**Discussion by erind7998** [community/community#61770] [Codespaces]
### Select Topic Area

Show & Tell

### Body

https://github.com/community/community/assets/136864277/f369ee97-bd57-4d7c-8d3c-a0fe3b61fad3

How to make this script a actual video?

**Comment by GhostOf0days** [comment]
Copy and paste the URL into your search bar and hit enter.

**Reply by kurotsmile** [reply]
love it

**Reply by erind7998** [reply]
love what?
